main/src/ca/uwaterloo/flix/language/phase/unification/Unification.scala

```diff
@@ -144,7 +144,7 @@ object Unification {
         } else {
           // Case 1.2: The fields do not match, attempt to match with a field further down.
           visit(restRow2) map {
-            case (subst, rewrittenRow) => (subst, Type.mkRecordExtend(field2, fieldType2, rewrittenRow))
+            case (subst, rewrittenRow) => (subst, Type.mkRecordExtend(field2, fieldType2, rewrittenRow, SourceLocation.Unknown))
```

Comment:
We should have access to a lot of source locs here! (they are just underscored...)